src/ralph/export_to_ng/resources.py

```diff
@@ -429,6 +430,16 @@ def get_queryset(self):
 
 class NetworkResource(resources.ModelResource):
```

Comment:
I'd add here `excluded_networks = []` property if somehow this resource is called outside of mgmt command